Real_Masters_all/mdaily.xml

```diff
@@ -37648,7 +37649,8 @@
               <container type="box" label="Box">40</container>
               <unittitle>"Photo Archive <unitdate type="inclusive" normal="1993/1994">1993-1994</unitdate> Copy</unittitle>
               <physdesc>
-                <physfacet>CD R, black and white and color images in various formats, 1994 images unreadable</physfacet>
+                <extent>1 optical disks</extent>
+                <physfacet>CD-R</physfacet>
```

Comment:
I could put the lost info in ```<odd>``` tags for sure (I assume that's what you were referring to - github ate the original tag).